math/clingo/Portfile

```
@@ -36,6 +39,11 @@ configure.args-append \
                     -DCLINGO_BUILD_WITH_PYTHON=OFF \
                     -DCLINGO_BUILD_WITH_LUA=OFF
 
+if {${build_arch} eq "ppc" || ${build_arch} eq "ppc64" } {
```

This is unlikely to be correct when using the unlversal portgroup. This block will take effect when the *build machine* is ppc or ppc64 (whether building for ppc or ppc64 or i386) and will not take effect when the build machine is not ppc nor ppc64 (whether building for ppc or ppc64 or i386). You probably want to have this apply when building *for* ppc or ppc64, which the muniversal portgroup gives you the facility to implement, at least with regard to flags. Patches must be applied unconditionally, which they should be anyway.

---

In the rare cases where you really do only want to check the build arch (for example in a port which does not have a universal variant) you can be more succinct about it using the Tcl `in` (in) and `ni` (not in) operators:
```suggestion
if {${build_arch} in [list ppc ppc64]} {
```

---

> This block will take effect when the _build machine_ is ppc or ppc64

Well, to be completely precise, it will take effect when the single-architecture build would be for ppc or ppc64, which is not applicable if the universal variant is selected. The point is: this port will do something different when building universal for ppc and i386 on a PowerPC Mac vs. when building universal for ppc and i386 on an Intel Mac, which is almost certainly not what you intended to happen.

---

okay, sorry @ryandesign: I clearly don't understand this business well enough... I'll not be merging PRs anymore that deal with the "muniversal" PG and/or fixes for specific old systems that I have no way of testing. 

It would be good though if we could recruit some interested parties to help move the PR queue along... as PRs start to pile up without much response from members.